docs/guides/development/nodejs/nodejs-twitter-bot/index.md

```diff
@@ -0,0 +1,294 @@
+```YAML
+author:
+     name: Pj Metz
+     email: metz.pj@gmail.com
+description: 'Building a new Twitter bot and hosting it on Linode. Guide is for beginners new to coding and cloud services.'
+keywords: ["node.js", "beginner", "javascript", "twitter bot"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2021-04-27
+modified_by:
+	name: Linode
+title: 'Node.js Twitter Bot Beginner’s Guide: Replying to Users and Scheduling Tweets'
+contributor:
+	name: Pj Metz
+	link: [Metzinaround.com](https://www.metzinaround.com)
+external_resources:
+	- "[Creating a new User in Ubuntu 		 (https://youtu.be/fDHHKR0nVQg)"
+	- "[FileZilla](https://www.linode.com/docs/guides/filezilla/)
+```
+
+#  Node.js Twitter Bot Beginner’s Guide: Replying to Users and Scheduling Tweets
+
+Twitter bots are a great way to practice using an API and developing some coding skills along the way. You can usually get the bot going for under 100 lines of code, so it’s especially good for beginners. In this tutorial, I’ll show you how to use Node.js, *a framework for javascript* and a few npms, or *node package managers* to have your bot tweeting in no time.
+
+## Before you Begin:
+
+You should download or install:
+[Visual Studio Code](https://code.visualstudio.com/)
+[GitHub desktop](https://desktop.github.com/)
+
+Be ready to visit:
+[Twitter](https://twitter.com/home)
+[Twitter Developer portal](https://developer.twitter.com/en)
+[GitHub](https://github.com/)
+[Linode](https://www.linode.com)
+
+You can look at the documentation:
```

Comment:
I moved these documentation links down into the part of the guide where the corresponding packages are installed with npm